docs/tutorial5.rst

```diff
@@ -0,0 +1,508 @@
+        coordinates=desired_output_points,
+        output_parameters=JsonOutputParameters(
+            output_interval=delta_time,
+            nodal_results=nodal_results,
+            gauss_point_results=gauss_point_results
+        )
+    )
+
+    # END CODE BLOCK
+
+
+Now that the first stage of the model is set up, the Stem class needs to be  initialised,
+with the model and the directory where the input files will be written to.
+
+.. code-block:: python
+
+    input_files_dir = "compute_train_with_joint"
+    stem = Stem(model, input_files_dir)
+
+    # END CODE BLOCK
+
+The second stage can easily be created  by calling the "create_new_stage" function.
+This copies the entire stage into stage 2. The new stage requires the definition of a duration and a time step.
+
+.. code-block:: python
+
+    delta_time_stage_2 = 1e-3
+    duration_stage_2 = 0.5
+    stage2 = stem.create_new_stage(delta_time_stage_2, duration_stage_2)
+
+    # END CODE BLOCK
+
+In the second stage we want to compute the dynamic response of the moving train.
+Therefore, the solution type needs to be set to "DYNAMIC" and the Rayleigh damping coefficients adjusted
+(0.0002 for the stiffness matrix and 0.6 for the mass matrix).
+Since the problem is linear elastic, the Linear-Newton-Raphson strategy is used.
+The train velocity also needs to be adjusted to 40 m/s. This adjustment needs to be done on the train load parameters
+and on the UVEC parameters.
+The static initialisation in the UVEC, needs to be set to False to model the dynamic behaviour of the vehicle.
+
+.. code-block:: python
+
+    velocity = 40
+    stage2.project_parameters.settings.solution_type = SolutionType.DYNAMIC
+    stage2.project_parameters.settings.strategy_type = LinearNewtonRaphsonStrategy()
+    stage2.project_parameters.settings.rayleigh_k = 0.0002
+    stage2.project_parameters.settings.rayleigh_m = 0.6
+    stage2.get_model_part_by_name("train_load").parameters.velocity = velocity
+    stage2.get_model_part_by_name("train_load").parameters.uvec_parameters["velocity"] = velocity
+    stage2.get_model_part_by_name("train_load").parameters.uvec_parameters["static_initialisation"] = False
+
+    # END CODE BLOCK
+
+After the stage is created, and the settings are set, the stage is added to the calculation.
+The calculation is then ran by calling the run_calculation function within the stem class.
+
+.. code-block:: python
+
+    stem.add_calculation_stage(stage2)
+    stem.write_all_input_files()
+    stem.run_calculation()
+
+    # END CODE BLOCK
+
+Once the calculation is finished, the results can be visualised using Paraview, or by loading the json output file.
+
+This figure shows the results for the three nodes that were defined in the model.
+The figure compares the results of the model with rail joint and without rail joint.
+The analysis for this results have been obtained with an element size of 0.25m, time step of 5e-4s and a
+duration of 5e-3s for stage 1 and 0.5s for stage 2.
```

Comment:
But for the sake of a quick tutorial,  the element size and time step in this tutorial are set to coarse values.